### PR TITLE
Kw-add all fonts for the app

### PR DESCRIPTION
**Pull request by KanikaWadhawan**
# add all fonts for the app

### CODE REVIEW

File: client/js/assets/fonts/stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+    font-family: 'Neue Haas Grotesk Display Pro';
+    src: url('NeueHaasDisplay-XThinItalic.eot');
+    src: local('Neue Haas Grotesk Display Pro 26 XThin Italic'), local('NeueHaasDisplay-XThinItalic'),
+        url('NeueHaasDisplay-XThinItalic.eot?#iefix') format('embedded-opentype'),
+        url('NeueHaasDisplay-XThinItalic.woff') format('woff'),
+        url('NeueHaasDisplay-XThinItalic.ttf') format('truetype');
+    font-weight: 100;
+    font-style: italic;
+}
+
```
Comment (siddharthparmar7) [inline client/js/assets/fonts/stylesheet.css]: What does this file do? If you are adding just fonts, you do not need this file.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Looks like your .DS_Store file is back. Please remove it. 

Why did you add fonts to the Android's assets dir?

File: client/js/assets/fonts/stylesheet.css (hunk 1 above)
Comment (KanikaWadhawan) [reply client/js/assets/fonts/stylesheet.css]: Made the following changes. Thank you.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
.DS_Store file is still there, in root dir.  Please delete it.

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**